{{ cookiecutter.__project_name_kebab_case }}/Dockerfile

```
@@ -75,6 +75,7 @@ RUN --mount=type=cache,target=/var/cache/apt/ \
     sh -c "$(curl -fsSL https://starship.rs/install.sh)" -- "--yes" && \
     usermod --shell /usr/bin/zsh user && \
     echo 'user ALL=(root) NOPASSWD:ALL' > /etc/sudoers.d/user && chmod 0440 /etc/sudoers.d/user
+RUN  git config --system --add safe.directory '*'
```

Sorry about the pedantry, but there's an superfluous space here that I'd prefer not to have for consistency with other RUN commands:
```suggestion
RUN git config --system --add safe.directory '*'
```

---

Thanks for noticing! I've adapted the PR :)